src/jvmTest/kotlin/test.kt

```diff
@@ -172,6 +172,7 @@ internal class DiamondTests {
     @ParameterizedTest(name = "{0}")
     @MethodSource("getAllFilenames")
     fun test(filepath: String) {
+        if (filepath.contains("benchmark")) return
```

Comment:
👍 ad-hocs are what we do love

Consider NOT including this in commits. You can disable benchmarks locally (even with such adhoc), or you can consider filing a separate PR.